keyboards/keychron/k2/rgb/v2/ansi/info.json

```diff
@@ -0,0 +1,269 @@
+{
+    "keyboard_name": "Keychron K2v2 RGB ANSI",
+    "manufacturer": "Keychron",
+    "url": "https://www.keychron.com/products/keychron-k2-wireless-mechanical-keyboard",
+    "maintainer": "ITNerd69",
+    "usb": {
+        "pid": "0xFE02",
+		"vid": "0x0001",
+		"device_version": "1.0.0"
+    },
+	"matrix_pins": {
+        "cols": ["A8", "A9", "A10", "A11", "A12", "A13", "A14", "A15", "B0", "B1", "B2", "B3", "B4", "B5", "B6", "B7"],
+        "rows": ["C15", "D11", "D10", "D9", "D8", "D7"]
+    },
+    "diode_direction": "COL2ROW",
+	"features": {
+        "rgb_matrix": true,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": true,
+        "dip_switch": true
+    },
+    "processor": "SN32F248BF",
+    "bootloader": "sn32-dfu",
+	"debounce": 5,
+	"indicators": {
+        "caps_lock": "B9",
+        "on_state": 1
+    },
+   "width": 16,
+   "height": 6,
```

Comment:
probably not needed, since they get calculated from matrix size.

Reply:
Removed.